server/controllers/usersControllers.js

```diff
@@ -1,13 +1,21 @@
 import jwt from 'jsonwebtoken';
 import bcrypt from 'bcrypt';
-import _ from 'lodash';
 import validate from '../middlewares/validateUser';
 import validateLogin from '../middlewares/validateLogin';
-import users from '../models/usersModels';
 import express from 'express';
 import response from '../helpers/response';
 const router = express.Router();
 
+import {Client, Pool} from 'pg';
+import dotenv from 'dotenv';
+dotenv.config();
+const{JWT} = process.env;
```

Comment:
This code is similar to bookingsController.js make reusable methods instead of repeating code